Library/Formula/tmux.rb

```diff
@@ -43,14 +43,26 @@ def install
 
     system "make", "install"
 
-    bash_completion.install "examples/bash_completion_tmux.sh" => "tmux"
-    pkgshare.install "examples"
+    if head?
```

Comment:
Oh, forgot we hadn't fixed that. It should be `build.head?` in `def install`, aye. I guess `head?` is fine in the caveats, although to be honest I'm not _completely_ sure we regularly accept caveats that only impact `head` or `devel`.